pkg/test/helpers.go

```diff
@@ -368,3 +373,7 @@ func (t *TestingRuntime) SetCluster(clusterKey string, cluster *v1.Cluster) {
 	t.RosaRuntime.Cluster = cluster
 	t.RosaRuntime.ClusterKey = clusterKey
 }
+
+func GetStringAddress(str string) *string {
```

Comment:
@marcolan018 Is this used?

Reply:
yes, it's used in cmd_test.go

Reply:
@marcolan018 Can you replace this with `aws.String` instead. We already use that widely throughout the code-base. 

cheers!

Rob

Reply:
updated, thanks for the remind